api/room/schema.py

```diff
@@ -183,7 +184,29 @@ def mutate(self, info, room_id, **kwargs):
         return DeleteRoom(room=exact_room)
 
 
+class UpdateFirebaseToken(graphene.Mutation):
+
+    class Arguments:
+        room_id = graphene.Int(required=True)
+        firebase_token = graphene.String()
```

Comment:
Great work @michael-basweti,

Please share your thoughts on adding `required=True` on the firebase_token variable.
I think it is necessary seeing as the mutation requires it to implement the change.

Reply:
> Great work @michael-basweti,
> 
> Please share your thoughts on adding `required=True` on the firebase_token variable.
> I think it is necessary seeing as the mutation requires it to implement the change.
I will include it, thanks for that view